pkg/apis/admissionregistration/v1/zz_generated.defaults.go

```diff
@@ -0,0 +1,87 @@
+// +build !ignore_autogenerated
+
+/*
```

Comment:
same here

Reply:
This is by design for current arktos_copyright.sh. all new added files only has arktos copyrights if we didn't  add new added files to hack/arktos_copyright_copied_k8s_files.